src/ui/components/UrlExportSettings.tsx

```diff
@@ -145,16 +146,17 @@ export const UrlExportSettings = () => {
       <Row fill>
         <Input
           type='text'
+          required
           pattern='\S+'
-          placeholder='your access token'
+          placeholder='Your access token'
           value={settings.accessToken}
           onChange={value => updateSettings(draft => { draft.accessToken = value })}
         />
       </Row>
       <Footer>
         <WebLink align='start' href='https://github.com/lukasoppermann/design-tokens#design-tokens'>Documentation</WebLink>
         <CancelButton />
-        <Button type='button' onClick={handleFormSubmit} autofocus>Save & Export</Button>
```

Comment:
I have tried this one out with an invalid token, invalid URL, token with not enough privileges and while offline (wifi turned off), comparing the released plugin (production version) and the my version. I couldn't find any problem... In all the listed situations I got toast errors with the appropriate message. Could you please provide some reproduction steps for the issues you've found?

I would like to focus on fixing the "no error" bug, rather than reverting this, mostly for UX purposes. The arguments are the following:

- In the current (production) implementation, although there's an `onSubmit` handler on the form, there is no way to submit on `enter` in an input field, because of some react quirks. It strictly needs a submit button within the form to call handler.

- The other problem is that when clicking the button, `handleFormSubmit` receives as `event.target` the actual button element. This causes `checkValidity()` to return `true`, even when the form is invalid (because buttons, as input elements, are always valid). This means the action is attempted (save/export) and not prevented and errors highlighted.

  - For example submitting without a GH Actions URL (or any other required input), instead of highlighting the error and allowing me to input the value, it closes the popup and displays a toast error with `🚨 No server url was provided, push aborted!`

Reply:
Oh, wow, thanks. I will check this out and let you know what I find. I'd love to get this fixed.

Reply:
Okay, so if I add the `required` and change to `submit` removing the `onClick={handleFormSubmit}`

If I set the url to `https://test.com` and click the button or press return, I get the blank screen issue. Any idea?

Reply:
Just add `event.preventDefault()` in the `handleSubmit` function.
Otherwise the form will make the Figma browser to navigate, and since it's not a real web app, will display a blank screen.